File: LeetCode/[E] - 1732. Find the Highest Altitude/solution.py
class Solution:
    def largestAltitude(self, gain: list[int]) -> int:
        mxAlt, curr = 0, 0
        for i in range(len(gain)):
            curr += gain[i]
            if curr > mxAlt:
                mxAlt = curr
        return mxAlt

File: LeetCode/[E] - 1732. Find the Highest Altitude/solution.go
package main

func largestAltitude(gain []int) int {
	mxAlt, curr := 0, 0
	for i := 0; i < len(gain); i++ {
		curr += gain[i]
		if curr > mxAlt {
			mxAlt = curr
		}
	}
	return mxAlt
}
